### ISSUE TITLE
Version 1.4.3 has a breaking change for GHC 8.0.2

### ISSUE THREAD

**Issue by wenkokke** [hanshoglund/monadplus#7]
Would it make sense to release a patch release fixing this problem, and deprecating 1.4.3 on Hackage?

```
src/Control/Monad/Plus.hs:270:15: error:
    Ambiguous occurrence ‘<>’
    It could refer to either ‘Sem.<>’,
                             imported from ‘Data.Semigroup’ at src/Control/Monad/Plus.hs:67:1-28
                          or ‘Data.Monoid.<>’,
                             imported from ‘Data.Monoid’ at src/Control/Monad/Plus.hs:68:1-18
```

See https://github.com/agda/agda/issues/6436

**Comment by andreasabel** [comment]
Actually, it is not a breaking change (in the sense of the Haskell PVP), it is just a change breaking compilation with GHC <= 8.2:
- #8 

I think this issue can be closed in favor of #8.

**Comment by ocharles** [comment]
I believe this is fixed in https://github.com/hanshoglund/monadplus/commit/592f52f614f9e815de3214525ef2d3ebd4d20d01
